src/main/scala/com/microsoft/hyperspace/index/IndexLogEntry.scala

```diff
@@ -455,10 +490,40 @@ case class IndexLogEntry(
 
   def hasLineageColumn(spark: SparkSession): Boolean = {
     ResolverUtils
-      .resolve(spark, IndexConstants.DATA_FILE_NAME_COLUMN, schema.fieldNames)
+      .resolve(spark, IndexConstants.DATA_FILE_NAME_ID, schema.fieldNames)
       .isDefined
```

Comment:
Can we just change this to:
```scala
  def hasLineageColumn: Boolean = schema.fieldNames.contains(IndexConstants.DATA_FILE_NAME_ID)
```
since we don't need to resolve the internal column (should always exact match)? Mentioned in #181